Sources/PhoenixLiveViewNative/LiveViewCoordinator.swift

```diff
@@ -343,7 +343,9 @@ public class LiveViewCoordinator<R: CustomRegistry>: ObservableObject {
             if case .awaitingSocketConnection(let continuation) = self.internalState {
                 continuation.resume(throwing: Error.socketError(error))
             } else {
-                self.internalState = .connectionFailed(.socketError(error))
+                DispatchQueue.main.async {
```

Comment:
Use the new Swift Concurrency style of switching actors, to match the rest of the file:

```suggestion
                Task { @MainActor in
```

Reply:
Oh, yeah. New Swift Concurrency. I know all about that, for sure. Haven't been living under a rock at all.

After all, people have paid me for projects I've written in Swift in the past, so I'm the Sort Of Person Who Probably Knows About New Swift Concurrency.

/me googles for 3 minutes

Seems nice.

So, those changes I did can probably be replaced by just this, and that same thing on line 443. Keep everything on the main thread in the first place, which we can afford to do, and rm the rest of the changes 👍. Will test that out.